src/Example/Models/User.cs

```diff
@@ -9,6 +9,7 @@ public User()
         public int Id { get; set; }
         public string YotiId { get; set; }
         public byte[] Photo { get; set; }
+        public string Base64Photo { get; set; }
```

Comment:
This should have been in the next commit, but I'll leave it there for now, as it doesn't break anything